doc/sphinx/proofs/writing-proofs/inductive.rst

```diff
@@ -779,29 +774,30 @@ and an explanation of the underlying technique.
    Let :n:`{+ @ident}` be identifiers in the local context. This tactic behaves
    as generalizing :n:`{+ @ident}`, and then performing ``inversion_clear``.
 
-.. tacv:: inversion_clear @ident as @or_and_intropattern_loc in {+ @ident}
+.. tacv:: inversion_clear @ident @as_or_and_ipat in {+ @ident}
 
    This allows naming the hypotheses introduced in the context by
    :n:`inversion_clear @ident in {+ @ident}`.
 
-.. tacv:: dependent inversion @ident
-   :name: dependent inversion
+.. todo PR: expand "dependent" tactic in editing
+
+.. tacn:: dependent {| simple inversion | inversion | inversion_clear } {| @ident | @natural } {? @as_or_and_ipat } {? with @one_term }
```

Comment:
Q1: I suppose the options `simple inversion`, `inversion` and `inversion_clear` should be presented separately?  (The mlg file has them combined like this).

Q2: There are many tactics that contain "dependent" and "simple" (listed below).  Is there any commonality between the various "dependent" tactics or between the "simple" tactics, or do they mean different things in different tactics?

```
simple destruct {| @ident | @natural }
simple induction {| @ident | @natural }
simple injection {? @destruction_arg }
simple inversion {| @ident | @natural } {? @as_or_and_ipat } {? in {+ @ident } }
dependent induction @ident {? {| generalizing | in } {+ @ident } } {? using @one_term }
dependent {| simple inversion | inversion | inversion_clear }
     {| @ident | @natural } {? @as_or_and_ipat } {? with @one_term }
```

Reply:
Q1: yes, I guess this would be more readable if they could be separated.

Q2: yes, I think there are some commonalities, although I couldn't explain precisely, because I do not use any of these `simple` / `dependent` variants.

Reply:
No idea on Q1, but for Q2, I think (though I'm not sure) that "simple" and "dependent" share roughly the same meanings across these tactics (though `simple refine`, for example, is very different).